lib/openshift/network.rb

```diff
@@ -3,18 +3,13 @@
 module BushSlicer
   # @note represents an OpenShift environment Network class
 
-  class NetworkOperator < ClusterResource
-    # full network.operator.openshift.io name doesn't seem to work,
-    # RuntimeError: hash not from a NetworkOperator: expected  but was Network
-    # raise "hash not from a #{shortclass}: expected #{kind} but was #{hash["kind"]}"
-    # api_name_no_dots = "operatoropenshiftio"
-    # resource_without_dots = "networksoperatoropenshiftio"
-    # lib/openshift/config_imageregistry_operator_openshift_io.rb` - for objects without unique name and you need full qualified name
+  class Network < ClusterResource
     RESOURCE = "network.operator"
 
     def network_type(user: nil, cached: true, quiet: false)
       rr = raw_resource(user: user, cached: cached, quiet: quiet)
-      rr.dig('spec', 'defaultNetwork', 'type')
+      return rr.dig('spec', 'defaultNetwork', 'type') || \
+             rr.dig('spec', 'networkType')
```

Comment:
network.operator do not have `networkType`

Reply:
Misread `network.operator.openshift.io` with `network.config.openshift.io`, update the PR accordingly.